micrometer-api/src/main/java/io/micrometer/api/instrument/observation/Observation.java

```diff
@@ -397,6 +390,15 @@ public void remove(Object key) {
             return (T) this.map.get(key);
         }
 
+        /**
+         * Removes an entry from the context.
+         *
+         * @param key key by which to remove an entry
+         */
+        public void remove(Object key) {
```

Comment:
Maybe we should return the removed object?

Reply:
👍🏼 I just moved this method, did not change the signature but we should definitely do this.